Index: modemmanager-1.4.12/plugins/dell/mm-plugin-dell.c
===================================================================
--- modemmanager-1.4.12.orig/plugins/dell/mm-plugin-dell.c	2016-04-08 11:16:22.000000000 +0800
+++ modemmanager-1.4.12/plugins/dell/mm-plugin-dell.c	2016-04-15 10:21:39.880746314 +0800
@@ -297,7 +297,7 @@
 {
     CustomInitContext *ctx;
 
-    ctx = g_slice_new (CustomInitContext);
+    ctx = g_slice_new0 (CustomInitContext);
     ctx->result = g_simple_async_result_new (G_OBJECT (probe),
                                              callback,
                                              user_data,
@@ -306,6 +306,7 @@
     ctx->port = g_object_ref (port);
     ctx->cancellable = cancellable ? g_object_ref (cancellable) : NULL;
     ctx->gmi_retries = 3;
+    ctx->cgmi_retries = 3;
     ctx->ati_retries = 3;
 
     custom_init_step (ctx);
